std/uuid.d

```diff
@@ -148,6 +148,12 @@ public struct UUID
                 179, 189, 251, 70])._toString() == "8ab3060e-2cba-4f23-b74c-b52db3bdfb46");
         }
 
+        // Reinterpret the UUID as an array of some other primitive.
+        @trusted ref T[16 / T.sizeof] asArrayOf(T)()
```

Comment:
Is this "return" a new function attribute or something? I've also seen mentions of `@return` too but have been unable to find any info on it, including in the changelogs. Can someone explain? Sorry for being out of the loop...